IframeMediaPause/iframe_media_pausing.md

```diff
@@ -0,0 +1,189 @@
+# Pausing iframe media when not rendered
+
+Authors: [Gabriel Brito](https://github.com/gabrielsanbrito), [Steve Becker](https://github.com/SteveBeckerMSFT), [Sunggook Chue](https://github.com/sunggook), Ravikiran Ramachandra
+
+## Status of this Document
+This document is a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
+* This document status: **`ACTIVE`**
+* Expected venue: [Web Hypertext Application Technology Working Group (WHATWG)](https://whatwg.org/)
+* Current version: **https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/IframeMediaPause/iframe_media_pausing.md**
+
+## Introduction
+
```

Comment:
Can we front-load our description of what the proposal aims to do? e.g. start out with something like this:
`
Web applications that host embedded media content via iframes may wish to respond to application input by temporarily hiding the media content. These applications may not want to unload the entire iframe when it's not visible since it could generate user-perceptible performance and experience issues when showing the media content again. At the same time, the user could have a negative experience if the media continues to play and emit audio when not visible. This proposal aims to provide web applications with the ability to control embedded media content in such a way that guarantees their users have a good experience when the iframe's visibility is changed.
`

Reply:
Adopted your proposal but switching uses of "visibility" for "render". Thanks!